interfaces/builtin/uio.go

```diff
@@ -100,6 +103,19 @@ func (iface *uioInterface) AppArmorConnectedPlug(spec *apparmor.Specification, p
 	// device node in /dev. Use AddDeduplicatedSnippet() for clarity
 	// in the resulting rules.
 	spec.AddDeduplicatedSnippet("/sys/devices/platform/**/uio/uio[0-9]** r,  # common rule for all uio connections")
+	uioConfigPath := "/sys/class/uio/" + iface.getUIOName(path) + "/device/config"
```

Comment:
`iface.getUIOName(path)` can be simply `strings.TrimPrefix(path, "/dev/")` (path has been already verified with a regex so it should be fine)

Reply:
done